share/completions/clean.fish

```diff
@@ -0,0 +1 @@
+complete --command clean --no-files --arguments 'all\t"Specify that each and every sector on the disk is set to zero"'
```

Comment:
This file is missing a new line, which is not usually an issue but occasionally trips things up. Rather than an embedded tab I suggest using the `--description`option.